src/main/java/de/hpi/bpt/argos/util/RestEndpointUtilImpl.java

```diff
@@ -145,8 +146,19 @@ public String getParameter(String parameterName, boolean includePrefix) {
 	 */
 	@Override
 	public String executeRequest(Logger logger, Request request, Response response, Route route) {
+		long startMs = System.currentTimeMillis();
 		logReceivedRequest(logger, request);
 
+		WatchImpl.measure(String.format("execute request: [%1$s] '%2$s'", request.requestMethod(), request.uri()),
+				() -> executeRequest(request,response, route));
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') ',' is not followed by whitespace. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=checkstyle%3AWhitespace_After_BP2016)